appveyor.yml

```diff
@@ -116,6 +116,7 @@ before_deploy:
   - "%CMD_IN_ENV% python.exe -c \"import sys; print(sys.version)\""
   - "%CMD_IN_ENV% python.exe setup.py recythonize"
   - "%CMD_IN_ENV% python.exe setup.py install"
+  - "%CMD_IN_ENV% pip install --upgrade hyperspyui"
```

Comment:
It seems the `--upgrade` flag is causing some problems: https://ci.appveyor.com/project/vidartf/hyperspy/build/1.2.0.358/job/atqrbqwvka3b1k4c